manufacturing/enclosure/README.md

```diff
@@ -46,6 +27,25 @@ BEFORE purchasing any parts.**
 
 **Total assembly price:** USD 430.90
 
+----
+## Custom parts
+
+
+
+### Fabrication Drawings
+
+|  |   |
+|---|---|
+|![](images/fab_drawing_2.jpg) |![](images/fab_drawing_3.jpg)|
+|![](images/fab_drawing_4.jpg) |![](images/fab_drawing_5.jpg) |
+
+The ventilator enclosure is made up a folded, 1.52mm thick, stainless steel sheet.
+
+| File Type  | Link |
+|:------|:-------|
+| Enclosure Fabrication CAD | [.ipt](enclosure.ipt) |
+| Enclosure Fabrication Drawing | [.pdf](enclosure.pdf)|
+
```

Comment:
The "how much sheet metal?" line belongs with the BOM section, but has been pushed down here.